srcpkgs/linux5.13-surface/template

```diff
@@ -0,0 +1,303 @@
+# Template file for 'linux5.13-surface'
+pkgname=linux5.13-surface
+version=5.13.13
+revision=3
```

Comment:
Revision must be independent from any external resources. Use version=5.13.13.3, `arch-${version%.*}-${version##*.}`, set revision to 1.

Reply:
Done, set revision to `1` and version to `5.13.13`
Using a static link to the linux-surface distfile